guides/common/modules/con_host-substatus-overview.adoc

```diff
@@ -1,55 +1,271 @@
+* *No changes*: During the last configuration, nothing changed.
+* *No reports*: This can be both a *Warning* or *OK* status.
+When there are no reports but the host uses an associated {SmartProxy} for configuration management or the `always_show_configuration_status` setting is set to `true`, it maps to *Warning*.
+Otherwise it maps to *OK*.
+* *Error*: This indicates an error during configuration.
+For example, a configuration run failed to install a package.
+* *Out of sync*: A configuration report was not received within the expected interval, based on the `outofsync_interval` setting.
+Reports are identified by an origin and can have different intervals based upon it.
 
-In the *Properties* table of the host details' page, you can view both the global host status and all sub-statuses.
+Build::
+This sub-status is only relevant for hosts provisioned from {Project} or hosts registered through global registration.
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| Installed | OK | 0
+| Pending installation | OK | 1
+| Token expired | Error | 2
+| Installation error | Error | 3
+|===
 
-Each sub-status can define its own set of possible values that are mapped to the three global status values.
+ifndef::foreman-deb[]
+Compliance::
+Indicates if the host is compliant with OpenSCAP policies.
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| Compliant | OK | 0
+| Inconclusive | Warning | 1
+| At least one incompliant | Error | 2
+|===
+endif::[]
 
-The *Build* sub-status has two possible values {endash} *pending* and *built* that are both mapped to global *OK* value.
+ifdef::satellite,orcharhino[]
+OVAL scan::
+Indicates if there are any vulnerabilities found on the host
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| No vulnerabilities found | OK | 0
+| Vulnerabilities found | Warning | 1
+| Vulnerabilities with available patch found | Error | 2
+|===
+endif::[]
 
-The *Configuration* status has more possible values that map to the global status as follows:
+Execution::
+Status of the last completed remote excution job.
+ifdef::foreman-el,foreman-deb[]
++
+Only applies if you have the Remote Execution plug-in installed.
+endif::[]
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| Last execution succeeded / No execution finished yet | OK | 0
+| Last execution failed | Error | 1
+| Unknown execution status | OK | 2 or 3
+| Last execution cancelled | OK | 4
+|===
 
-.Sub-statuses that map to the global *OK* status
+ifdef::satellite,orcharhino,katello[]
+Inventory::
+Indicates if the host is synchronized to {RHCloud}.
+{ProjectServer} performs the synchronization itself but only uploads basic information to {RHCloud}.
++
```

Comment:
```suggestion
+
ifdef::orcharhino[]
+
Only applies if you have the Red Hat Cloud plug-in installed.
endif::[]
```

See https://docs.theforeman.org/3.9/Managing_Hosts/index-orcharhino.html#Installing_Red_Hat_Cloud_Plug_in_managing-hosts

You can also drop "katello" from line 108.

Reply:
Applied.